Packages/com.unity.inputsystem/InputSystem/Devices/Touchscreen.cs

```diff
@@ -507,6 +509,22 @@ public class Touchscreen : Pointer, IInputStateCallbackReceiver, IEventMerger, I
         /// </remarks>
         public TouchControl primaryTouch { get; protected set; }
 
+		/// <summary>
```

Comment:
Run formatting script ./format.ps1?
Suggest linking to https://docs.unity3d.com/ScriptReference/Display.html for correlation on how to use it.